app/controllers/events_controller.rb

```diff
@@ -0,0 +1,55 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show]
+  before_action :signed_in?, only: [:new]
+
+  # GET /events
+  # GET /events.json
+  def index
+    @events = Event.all
+  end
+
+  # GET /events/1
+  # GET /events/1.json
+  def show
+    @users = User.all
+  end
+
+  # GET /events/new
+  def new
+    @event = Event.new
+  end
+
+  # POST /events
+  # POST /events.json
+  def create
+    @event = Event.new(event_params)
+    @event.creator_id = session[:user_id]
```

Comment:
You should use the  `#build` method on the association,
i.e. `@event = current_user.events.build(event_params)` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

Note that `current_user` is a helper method, which you have to declare in your helper module.